frontend/sw360-portlet/src/main/webapp/WEB-INF/tags/TrimLineBreaks.tag

```diff
@@ -0,0 +1,12 @@
+<%--
+  ~ Copyright Siemens AG, 2017. Part of the SW360 Portal Project.
+  ~
+  ~ All rights reserved. This program and the accompanying materials
+  ~ are made available under the terms of the Eclipse Public License v1.0
+  ~ which accompanies this distribution, and is available at
+  ~ http://www.eclipse.org/legal/epl-v10.html
+  ~
+  ~ This tag file MUST NOT have any line breaks outside of this multiline comment.
+  ~ It is surely possible to implement it in Java to avoid having this weird restriction,
+  ~ but this way was perceived as quickest by me at the time.
+  --%><%@ attribute name="input" type="java.lang.String" required="true" %><%@ taglib prefix="c" uri="http://java.sun.com/jstl/core_rt"%><%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %><c:set var="newline" value='<%="\n"%>' />${fn:replace(input, newline, '')}
```

Comment:
Is it necessary that there are no newlines in this file

The file ends without a trailing newline.